autosklearn/estimators.py

```diff
@@ -492,13 +492,72 @@ def score(self, X, y):
         return self.automl_.score(X, y)
 
     def show_models(self):
-        """Return a representation of the final ensemble found by auto-sklearn.
```

Comment:
I had to add the required text for `show_models()` here for it show up in the docs. Should I remove the same text I have added in `automl.py`?

Reply:
It's okay to duplicate it, it should probably not be long term but for now, I'd add it over to `automl.py` if the same function is there.